Modules/Segmentation/SuperPixel/test/itkSLICImageFilterTest.cxx

```diff
@@ -140,5 +148,6 @@ itkSLICImageFilterTest(int argc, char * argv[])
   }
 
 
-  return 0;
+  std::cout << "Test finished." << std::endl;
```

Comment:
I don't think this print statement above is useful

Reply:
This was brought in PR #3724 and maybe several other PRs as well; [memory was failing me](https://github.com/InsightSoftwareConsortium/ITK/pull/3581#discussion_r955884994) but [Dženan made it clear what I believe is one of the benefits of this] https://github.com/InsightSoftwareConsortium/ITK/pull/3581#discussion_r956041243. Eventually, using a macro that involves the message and the return code (cross-referencing issue #1273) could be beneficial, and bring the debate down to a single line across the whole codebase.

Reply:
Just add it here if it is important:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/TestKernel/include/itkTestDriverAfterTest.inc#L23